docs/src/domains/cdn/_index.md

```diff
@@ -1,183 +1,171 @@
-So it uses the origin name provided by Platform.sh.
-
-To ensure your TLS certificates are valid for both requests from clients to the CDN and from the CDN to the server on Platform.sh,
-you need to take two additional steps:
-
-1. Configure your CDN to set the `X-Forwarded-Host` HTTP header to your domain, for example: `example.com`.
-   That allows the request from the CDN to Platform.sh to still carry the original requested domain.
-   The specific way to do so varies by the CDN.
-2. Ensure your app can read from the `X-Forwarded-Host` header should it need the Host information.
-   Many popular apps already do so,
-   but if you have a custom app make sure that it checks for that header
-   and uses it instead of `Host` as appropriate.
-
-## Web Application Firewall & Anti-DDoS
-
-All Platform.sh-hosted sites, either Grid or Dedicated, live on infrastructure provided by major cloud vendors.
-These vendors include their own Level 3 DDoS protection that is sufficient for the majority of cases.
-
-Customers are welcome to put their own WAF in front of a Dedicated cluster or add other security measures not included in the offering.
-
-## The router cache
-
-When using a CDN the Platform.sh router's HTTP cache becomes redundant.
-In most cases it's best to disable it outright.
-Modify your route like so to disable the cache:
-
+ 
+Using a CDN speeds up the delivery of your site's content to its users.
+The CDN deploys servers at many locations around the world, known as Points of Presence (PoPs).
+Each PoP accommodates an edge server that behaves like a local cache to nearby users.
+Bringing content closer to users helps enhance your site's perceived performance,
+and can therefore improve user engagement and retention.
+ 
+By default, Dedicated projects include a [Fastly](./fastly.md) CDN managed by Platform.sh.
+Self-Service Grid plans don't include a CDN by default, but you can set up one at any time.
+
+## Before you begin
+
+You need:
+
+- An up-and-running Platform.sh project
+- A CDN such as [Fastly](./fastly.md) or [Cloudflare](./cloudflare.md)
+
+## Check your DDoS protection
+
+All sites hosted by Platform.sh live on infrastructure provided by major cloud providers.
+By default, you therefore benefit from the powerful DDoS protection services these cloud providers offer.
+In most cases, it's enough to keep your site safe from DDoS attacks when using a CDN.
+
+Note that Enterprise and Elite projects on Platform.sh also include a Web Application Firewall (WAF).
+However, depending on your needs, you can [install your own WAF](../../security/waf.md) 
+and add security measures not included in your plan.  
+
+## Create a CNAME record for your domain name
+
+To start routing client traffic through your CDN,
+create `CNAME` records for your domain names through your DNS provider.
+For more information, see you DNS provider's official documentation.
+
+Note that `CNAME` records can't point to apex domains.
```

Comment:
When digging on `CNAME` records, all the resources I came across stated that `CNAME` records can't point to apex domains, but that there are workarounds provided by DNS providers. So the assumption in our docs that a registrar might actually support 'CNAME' records for apex domains by default seems to be wrong. It seems it will always be through a workaround.